doggos/fuzzy_sets/interval_type2_fuzzy_set.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import annotations
+from typing import Callable, Tuple, NoReturn
+
+
+from doggos.fuzzy_sets.fuzzy_set import FuzzySet
+
+
+class IntervalType2FuzzySet(FuzzySet):
+    """
+    Class used to represent a fuzzy set type II :
```

Comment:
Change "a fuzzy set type II" to "an interval type II fuzzy set". Don't leave space before ':'.